internal/nodes/avi_model_advl4_translator.go

```diff
@@ -78,9 +79,18 @@ func (o *AviObjectGraph) ConstructAdvL4VsNode(gatewayName, namespace, key string
 		}
 	}
 
+	infraSetting, err := getL4InfraSetting(key, namespace, nil, &gw.Spec.Class)
```

Comment:
`infraSetting` is only getting used in L104. It will be better if we can move this just before L104.